# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserBridgeHost.h
# Commit: 1663719e2aa5
# Full Hash: 1663719e2aa5b6f77e32398c6d6fcb4cb3c999ed
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 15:55:11
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 16 - Cycle collect RemoteBrowser and support weak references in BrowserHost. r=nika
#   
#   BrowserParent is cycle collected and supported weak references, so this commit adds support
#   for these things to BrowserHost.
#   
# ==============================================================================

diff -r 53663cad8890 -r 1663719e2aa5 dom/ipc/BrowserBridgeHost.h
--- a/dom/ipc/BrowserBridgeHost.h	Wed May 08 16:12:26 2019 -0500
+++ b/dom/ipc/BrowserBridgeHost.h	Tue May 14 16:28:10 2019 -0500
@@ -28,7 +28,8 @@
 
   explicit BrowserBridgeHost(BrowserBridgeChild* aChild);
 
-  NS_INLINE_DECL_REFCOUNTING(BrowserBridgeHost, override);
+  NS_DECL_CYCLE_COLLECTING_ISUPPORTS
+  NS_DECL_CYCLE_COLLECTION_CLASS(BrowserBridgeHost)
 
   // Get the IPDL actor for the BrowserBridgeChild.
   BrowserBridgeChild* GetActor() { return mBridge; }